.gitignore

```diff
@@ -0,0 +1 @@
+/target/
```

Comment:
you should probably add .project, .classpath, and .settings/ here to avoid these files getting included again.